flepimop/gempyor_pkg/tests/statistics/test_statistic_class.py

```diff
@@ -228,11 +228,95 @@ def simple_valid_resample_and_scale_factory() -> MockStatisticInput:
     )
 
 
+def simple_valid_factory_with_pois() -> MockStatisticInput:
+    data_coords = {
+        "date": pd.date_range(date(2024, 1, 1), date(2024, 1, 10)),
+        "subpop": ["01", "02", "03"],
+    }
+    data_dim = [len(v) for v in data_coords.values()]
+    model_data = xr.Dataset(
+        data_vars={
+            "incidH": (
+                list(data_coords.keys()),
+                np.random.poisson(lam=20.0, size=data_dim),
+            ),
+            "incidD": (
+                list(data_coords.keys()),
+                np.random.poisson(lam=20.0, size=data_dim),
+            ),
+        },
+        coords=data_coords,
+    )
+    gt_data = xr.Dataset(
+        data_vars={
+            "incidH": (
+                list(data_coords.keys()),
+                np.random.poisson(lam=20.0, size=data_dim),
+            ),
+            "incidD": (
+                list(data_coords.keys()),
+                np.random.poisson(lam=20.0, size=data_dim),
+            ),
+        },
+        coords=data_coords,
+    )
+    return MockStatisticInput(
+        "total_hospitalizations",
+        {
+            "name": "sum_hospitalizations",
+            "sim_var": "incidH",
+            "data_var": "incidH",
+            "remove_na": True,
+            "add_one": True,
+            "likelihood": {"dist": "pois"},
+        },
+        model_data=model_data,
+        gt_data=gt_data,
+    )
+
+
+def simple_valid_factory_with_pois_with_some_zeros() -> MockStatisticInput:
+    mock_input = simple_valid_factory_with_pois()
+
+    mock_input.config["zero_to_one"] = True
+
+    mock_input.model_data["incidH"].loc[
```

Comment:
Much better re DRY.

However, should this line be incidD?

Additionally: I'm a bit unclear in general why we are testing having both incidD and incidH - seems like one would suffice?

Reply:
I've removed `incidD` and only use `incidH` now.